rayon-core/src/lib.rs

```diff
@@ -78,7 +78,7 @@ mod test;
 
 pub use self::join::{join, join_context};
 pub use self::registry::ThreadBuilder;
-pub use self::scope::{scope, Scope};
+pub use self::scope::{in_place_scope, scope, Scope};
```

Comment:
We should re-export this from the `rayon` crate too.

Reply:
Done